core/src/main/java/org/apache/iceberg/jdbc/JdbcCatalog.java

```diff
@@ -137,6 +137,10 @@ public void initialize(String name, Map<String, String> properties) {
       this.connections = new JdbcClientPool(uri, properties);
     }
 
+    String initializeTablesProp = properties.get(JdbcUtil.INIT_CATALOG_TABLES_PROPERTY);
+    if (initializeTablesProp != null) {
+      this.initializeCatalogTables = Boolean.parseBoolean(initializeTablesProp);
```

Comment:
nit: we can directly use 

this.initializeCatalogTables = PropertyUtil.propertyAsBoolean(JdbcUtil.INIT_CATALOG_TABLES_PROPERTY, initializeCatalogTables);

Reply:
+1

Reply:
+1